java/src/jmri/jmrix/mqtt/MqttSystemConnectionMemo.java

```diff
@@ -97,6 +98,53 @@ public MqttReporterManager getReporterManager() {
                 });
     }
 
+    public MqttThrottleManager getThrottleManager() {
+        if (getDisabled()) {
+            return null;
+        }
+        return (MqttThrottleManager) classObjectMap.computeIfAbsent(ThrottleManager.class,(Class<?> c) -> {
+                    MqttThrottleManager t = new MqttThrottleManager(this);
+                    t.setSendThrottleTopic(getMqttAdapter().getOptionState("16.3"));
+                    t.setRcvThrottleTopic(getMqttAdapter().getOptionState("16.5"));
+                    t.setSendDirectionTopic(getMqttAdapter().getOptionState("17.3"));
+                    t.setRcvDirectionTopic(getMqttAdapter().getOptionState("17.5"));
+                    t.setSendFunctionTopic(getMqttAdapter().getOptionState("18.3"));
+                    t.setRcvFunctionTopic(getMqttAdapter().getOptionState("18.5"));
+                    return t;
+                });
+    }
+
+    public MqttPowerManager getPowerManager() {
+        if (getDisabled()) {
+            return null;
+        }
+        return (MqttPowerManager) classObjectMap.computeIfAbsent(PowerManager.class,(Class<?> c) -> {
+                    MqttPowerManager t = new MqttPowerManager(this);
+                    t.setSendTopic(getMqttAdapter().getOptionState("20.3"));
+                    t.setRcvTopic(getMqttAdapter().getOptionState("20.5"));
+                    return t;
+                });
+    }
+
+    public MqttConsistManager getConsistManager() {
```

Comment:
Static analysis failes on this line due to that this method should be annotated with `@Override`. Just add `@Override` on the line before the method. See the lines 26-27 in this class.

Reply:
Fixed